tests/integ/test_dataframe.py

```diff
@@ -107,11 +107,11 @@
 @pytest.fixture(scope="module", autouse=True)
 def setup(session, resources_path, local_testing_mode):
     if not local_testing_mode:
-        test_files = TestFiles(resources_path)
         Utils.create_stage(session, tmp_stage_name, is_temporary=True)
```

Comment:
btw there is one change I made in my PR to turn create_stage/drop_stage into no-op for local testing, so that we can reduce some if-else check in the test code
```python
    @staticmethod
    def create_stage(session: "Session", name: str, is_temporary: bool = True):
        if isinstance(session._conn, MockServerConnection):
            # no-op in local testing
            return
        session._run_query(
            f"create or replace {'temporary' if is_temporary else ''} stage {quote_name(name)}"
        )

    @staticmethod
    def drop_stage(session: "Session", name: str):
        if isinstance(session._conn, MockServerConnection):
            # no-op in local testing
            return
        session._run_query(f"drop stage if exists {quote_name(name)}")
```

Reply:
Sounds good, if I merge later I will remove this if block.

Reply:
minor: ditto on removing `if not local_testing_mode:` check